### PR TITLE
Sponsor

### PR DESCRIPTION
**Pull request by longtv2222**
Sponsor section for Hack Your Learning website.

### CODE REVIEW

**Comment by eugene-lok** [comment]
Looks pretty good so far! Just some feedback:
- I think it'd be nice to add the background gradient we have in the Figma draft
- On a similar note, maybe the "Sponsor the Impact" button could change colors/respond once you hover over it, like you did with the sponsor logos

File: src/components/Sponsor.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import schulich from '../assets/schulich.png'; // Import schulich logo
+import HSBC from '../assets/HSBC.png';
+import zoo from '../assets/zoo.png';
+import '../stylesheets/Sponsor.scss';
+import HYLButton from './HYLButton';
+
+const Sponsor = () => (
+  <>
+    <div id="gradient">
```
Comment (lukesno) [inline src/components/Sponsor.jsx]: Just a small modification here; could you change "gradient" to the className and make "#sponsors" the id? (This is for routing purposes)

File: src/stylesheets/Sponsor.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .button_position {
+    text-align: center;
+    padding : 30px;
+  }
+
+  #gradient {
```
Comment (lukesno) [inline src/stylesheets/Sponsor.scss]: Will have to change this to .gradient with modification outlined above

Comment (lukesno) [reply src/stylesheets/Sponsor.scss]: Might actually be better to make a more specific className, since gradient is a bit generic

**Review by eugene-lok — DISMISSED** [review]
Nicely done!

**Review by PixelPoncho — APPROVED** [review]

**Pull request merged**